### ISSUE TITLE
Checkmarks do not work with Live Preview and SNW

### ISSUE THREAD

**Issue by gncorbin** [TfTHacker/obsidian42-strange-new-worlds#45]
I have Live Preview own which changes the markdown "- [ ]" to a checkbox that is clickable. When SNW is enabled, the checkbox is not clickable; I can go in and edit the markdown, "- [x]" and the live preview properly updates. But the checkbox is not clickable to check or uncheck. I am also on the default theme.

**Comment by TfTHacker** [comment]
I am sorry to hear this. Let us see if we can resolve the issue. Would you mind quickly doing this in an empty vault, just to make sure you can reproduce it without other plugins or CSS snippets or themes. Then we can go from there.

**Comment by gncorbin** [comment]
I set up a test vault. The only thing installed there is Obsidian-BRAT and SNW. Still having same issue.

**Comment by gncorbin** [comment]
One more note. The (mis)behavior does not occur on ipadOS. The bug is only on macOS for me.

**Comment by TfTHacker** [comment]
> I set up a test vault. The only thing installed there is Obsidian-BRAT and SNW. Still having same issue.

Thank you for doing this. Can you please do a video (loom, etc) so I can see what is happening? I wasn't able to reproduce it.

**Comment by gncorbin** [comment]
I've attached a video. I try to show what happens with and without SNW enabled.

https://user-images.githubusercontent.com/45463759/202209208-f832bf5c-b322-4460-8fc2-8aec6502879b.mp4

**Comment by TfTHacker** [comment]
So the good news is this for sure is a bug. I was able to reproduce it. It is an issue with the default theme when the pane is not wide. If you switch to the minimal theme I am guessing it will work for you.

I have reported the issue to Obsidian, hopefully, they can fix it in the default theme. Other plugins that use the gutter like mine have this issue, so I can't fix it in my plugin.

ALso, I noticed if I closed the left sidebar, even with the default theme it works. 

thank you for reporting this.
